packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -115,6 +121,10 @@ export type GetSelector<Component extends (props: any) => JSX.Element | undefine
 export const getSelector = <Component extends (props: any) => JSX.Element | undefined>(
   component: Component
 ): GetSelector<Component> => {
+  /* @conditional-compile-remove(PSTN-calls) */
+  if (component === HoldButton) {
```

Comment:
we have an if condition here as well as a switch statement below it, both checking for HoldButton
Is one of these redundant?